gemfiles/rails_5.1.gemfile

```diff
@@ -1,7 +1,10 @@
+# frozen_string_literal: true
+
 # This file was generated by Appraisal
 
-source "https://ezcater.jfrog.io/ezcater/api/gems/ezcater-gem-source"
+source "https://rubygems.org"
```

Comment:
Not sure what's causing this the generator to defaul to rubygems. Should we change this back to the ezcater source?

Reply:
This is fine. It is probably based on the top-level Gemfile. This is a public gem, so it should not require access to Artifactory gems.